### PR TITLE
feat: remove sitemap script from prepare step

### PR DESCRIPTION
**Pull request by just-toby**
## Description

we don't want the sitemap script to run every time someone runs `yarn install` - this removes it from the `prepare` script and adds it as a step to the main -> staging pipeline

### CODE REVIEW

**Review by mattbspector — APPROVED** [review]
good call

File: .github/workflows/2-deploy-to-staging.yml [hunk 1]
```diff
@@ -35,7 +35,7 @@ jobs:
           accountId: ${{ secrets.CLOUDFLARE_ACCOUNT_ID }}
           projectName: interface-staging
           directory: build
-          githubToken: ${{ secrets.GITHUB_TOKEN }}
+          gitHubToken: ${{ secrets.GITHUB_TOKEN }}
```
Comment (zzmp) [inline .github/workflows/2-deploy-to-staging.yml:38]: revert?

File: public/sitemap.xml [hunk 2]
```diff
@@ -322,4 +322,32 @@
   <url loc="https://app.uniswap.org/tokens/polygon/0xa9f37d84c856fda3812ad0519dad44fa0a3fe207" lastmod="2023-10-06T18:13:27.852Z" priority="0.8"/>
   <url loc="https://app.uniswap.org/tokens/polygon/0xb6a5ae40e79891e4deadad06c8a7ca47396df21c" lastmod="2023-10-06T18:13:27.852Z" priority="0.8"/>
   <url loc="https://app.uniswap.org/tokens/polygon/0x49a0400587a7f65072c87c4910449fdcc5c47242" lastmod="2023-10-06T18:13:27.852Z" priority="0.8"/>
+  <url loc="https://app.uniswap.org/tokens/ethereum/0xf951e335afb289353dc249e82926178eac7ded78" lastmod="2023-10-10T16:03:26.272Z" priority="0.8"/>
```
Comment (zzmp) [inline public/sitemap.xml:325]: you shouldn't be updating the sitemap as a part of this PR

**Review by zzmp — CHANGES_REQUESTED** [review]
lgtm but the sitemap changes are unnecessary and you changed the casing of githubToken

File: .github/workflows/2-deploy-to-staging.yml (hunk 1 above)
Comment (just-toby) [reply .github/workflows/2-deploy-to-staging.yml:38]: well this is an error i saw when i was looking at these files - see the PR description

Comment (zzmp) [reply .github/workflows/2-deploy-to-staging.yml:38]: don't change these even if your IDE, as it's risky and it works as-is

Comment (just-toby) [reply .github/workflows/2-deploy-to-staging.yml]: ok,, reverted these too!

**Review by zzmp — APPROVED** [review]

**Comment by Shegy1994** [comment]
good gob

**Pull request merged**